scripts/cm_test_all_sandia

```diff
@@ -365,6 +412,8 @@ if [ "$MACHINE" = "sems" ]; then
                "cuda/9.2 $CUDA9_MODULE_LIST $CUDA_BUILD_LIST $KOKKOS_PATH/bin/nvcc_wrapper $CUDA_WARNING_FLAGS"
     )
   fi
+  SPACK_CUDA_ARCH="+maxwell50" #use an old one
```

Comment:
Which sems machine has Maxwell GPUs to test? Just curious

Reply:
I don't know the specifics of any SEMS machine. I'm just guessing they have *at least* Maxwell 50.

Reply:
Was just curious. Ready for merge then?